docs/source/markdown/podman-container-inspect.1.md.in

```diff
@@ -20,46 +20,47 @@ The keys of the returned JSON can be used as the values for the --format flag (s
-| .OCIConfigPath       | Path to OCI config file (string)                   |
-| .OCIRuntime          | OCI runtime name (string)                          |
-| .Path                | Path to container command (string)                 |
-| .PidFile             | Path to file containing container PID (string)     |
-| .Pod                 | Parent pod (string)                                |
-| .ProcessLabel        | SELinux label of process (string)                  |
-| .ResolvConfPath      | Path to container's resolv.conf file (string)      |
-| .RestartCount        | Number of times container has been restarted (int) |
-| .Rootfs              | Container rootfs (string)                          |
-| .SizeRootFs          | Size of rootfs, in bytes [1]                       |
-| .SizeRw              | Size of upper (R/W) container layer, in bytes [1]  |
-| .State ...           | Container state info (struct)                      |
-| .StaticDir           | Path to container metadata dir (string)            |
+| **Placeholder**          | **Description**                                    |
+| ------------------------ | -------------------------------------------------- |
+| .AppArmorProfile         | AppArmor profile (string)                          |
+| .Args                    | Command-line arguments (array of strings)          |
+| .BoundingCaps            | Bounding capability set (array of strings)         |
+| .Config ...              | Structure with config info                         |
+| .ConmonPidFile           | Path to file containing conmon pid (string)        |
+| .Created                 | Container creation time (string, ISO3601)          |
+| .Dependencies            | Dependencies (array of strings)                    |
+| .Driver                  | Storage driver (string)                            |
+| .EffectiveCaps           | Effective capability set (array of strings)        |
+| .ExecIDs                 | Exec IDs (array of strings)                        |
+| .GraphDriver ...         | Further details of graph driver (struct)           |
+| .HostConfig ...          | Host config details (struct)                       |
+| .HostnamePath            | Path to file containing hostname (string)          |
+| .HostsPath               | Path to container /etc/hosts file (string)         |
+| .ID                      | Container ID (full 64-char hash)                   |
+| .Image                   | Container image ID (64-char hash)                  |
+| .ImageDigest             | Container image digest (sha256:+64-char hash)      |
+| .ImageName               | Container image name (string)                      |
+| .IsInfra                 | Is this an infra container? (string: true/false)   |
+| .IsService               | Is this a service container? (string: true/false)  |
+| .KubeExitCodePropagation | Service conatiner's Exit-code propagation (string) |
```

Comment:
D'oh, typo!

Reply:
Argh. I need to learn not to rely (only) on `range-diff`

Reply:
https://twitter.com/vlntnrthbrg/status/1481661376095469570

:-)